doc/install_windows.txt

```diff
@@ -1,633 +1,177 @@
-  ``C:\MinGW\msys\1.0`` or ``C:\mingw64\msys``) and add the following line at
-  the beginning (note that you may need to use e.g. Wordpad to edit this file,
-  since Notepad gets confused by Unix-style line breaks):
+.. code-block:: bash
 
-    .. code-block:: bash
+    pip install pydot-ng
 
-        set HOME=%USERPROFILE%
+.. _gpu_windows:
 
-- If you do not have them already, install the latest versions of
-  `Python 2.x <http://www.python.org/download/windows>`__ and
-  corresponding `NumPy <http://sourceforge.net/projects/numpy/files/>`__
-  then `SciPy <http://sourceforge.net/projects/scipy/files/>`__
-  packages (simply use the executable installers).
-  Note that there are currently no official 64 bit releases of NumPy and
-  SciPy, but you can find unofficial builds
-  `here <http://www.lfd.uci.edu/~gohlke/pythonlibs/>`__.
+Install and configure the GPU drivers (recommended)
+---------------------------------------------------
 
-- Ensure that the Python installation directory and its ``Scripts``
-  sub-directory are in your system path. This may be done by
-  modifying the global ``PATH`` Windows environment variables, or by creating
-  a ``.profile`` file in your MinGW home, containing a line like
-  ``export PATH=$PATH:/c/Python27:/c/Python27/Scripts`` (note that the latter
-  will work only when you run Theano from an MSYS shell).
+.. warning::
 
-- If you are installing the 64 bit version, you will need the following hack
-  to be able to compile Theano files with GCC (skip this step if you are using
-  the 32 bit version). In a temporary work directory, copy ``python27.dll``
-  (found in ``C:\\Windows\\System32``) as well as
-  `python27.def <http://wiki.cython.org/InstallingOnWindows?action=AttachFile&do=get&target=python27.def>`__.
-  Edit ``python27.def`` and replace ``Py_InitModule4`` with ``Py_InitModule4_64``.
-  Then open an MSYS shell, go to this temporary directory, and run:
-
-    .. code-block:: bash
-
-        dlltool --dllname python27.dll --def python27.def --output-lib libpython27.a
-
-  Finally, copy the libpython27.a file that was generated into your
-  ``C:\\Python27\\libs`` folder.
-
-- In order to run Theano's test-suite, you will need `nose
-  <http://nose.readthedocs.org/en/latest/>`__.
-  After unpacking its source code (you may use `7-zip
-  <http://www.7-zip.org/>`__), you can build and install it from within
-  its code directory by running the following command (either from a Windows
-  command prompt or an MSYS shell):
+    OpenCL support is still minimal for now.
 
-    .. code-block:: bash
-
-        python setup.py install
+Install CUDA drivers
+^^^^^^^^^^^^^^^^^^^^
 
+Follow `this link <https://developer.nvidia.com/cuda-downloads>`__
+to install the CUDA driver and the CUDA Toolkit.
 
+You must reboot the computer after the driver installation.
 
+Microsoft Visual C++ Compiler for Python 2.7 (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I don't think we should detail this.

If we are going to detail this make sure to make it very clear that this version of the C++ compiler is only for python 2.7.  Other python versions have a different requirement.